spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+require 'capybara/rails'
+
+RSpec.describe 'User index page', type: :feature do
+  describe 'when visiting the page' do
+    let!(:user1) { User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+
+    it 'shows the username of all other users' do
+      visit users_path
+      expect(page).to have_content(user1.name)
+    end
```

Comment:
- [ ] It would be better if we fix all failing test cases as we can see in the screenshot below
![Screenshot from 2023-04-19 19-00-23](https://user-images.githubusercontent.com/46388113/233174940-8d44bc09-17b3-421c-8066-88a922979afe.png)